src/components/mdpDatePicker/mdpDatePicker.js

```diff
@@ -423,15 +427,15 @@ module.directive("mdpDatePicker", ["$mdpDatePicker", "$timeout", function($mdpDa
                 };
 
                 ngModel.$validators.minDate = function(modelValue, viewValue) {
-                    return minDateValidator(viewValue, scope.dateFormat, scope.minDate);
+                    return minDateValidator(viewValue, scope.dateFormat, minDate);
```

Comment:
I think, here you break the possibility to change the `minDate` after creation of the picker using the scope binding. I believe we have to do a `scope.minDate || $mdpLocale.date.minDate` here (and cleanup the constant definition above).